packages/xchain-mayachain-amm/src/mayachain-amm.ts

```
@@ -439,6 +465,127 @@ export class MayachainAMM {
+  /**
+   * Add trade amount to account
+   * @param {AddToTradeAccountParams} param Add to trade account params
+   * @returns {TxSubmitted} Transaction made to add the trade amount
+   */
+  public async addToTradeAccount({ amount, address }: AddToTradeAccountParams): Promise<TxSubmitted> {
+    const quote = await this.estimateAddToTradeAccount({ amount, address })
+
+    if (!quote.allowed) throw Error(`Can not add to trade account. ${quote.errors.join(' ')}`)
+
+    // Convert trade asset to underlying asset for the transaction
+    const underlyingAsset = { ...quote.value.asset, type: quote.value.asset.type === AssetType.TRADE ? AssetType.NATIVE : quote.value.asset.type }
```

ESLint

[error] Replace `·...quote.value.asset,·type:·quote.value.asset.type·===·AssetType.TRADE·?·AssetType.NATIVE·:·quote.value.asset.type` with `⏎······...quote.value.asset,⏎······type:·quote.value.asset.type·===·AssetType.TRADE·?·AssetType.NATIVE·:·quote.value.asset.type,⏎···`

(prettier/prettier)